tests/components/rfxtrx/test_init.py

```diff
@@ -204,6 +204,7 @@ async def test_connect_failed(
 async def test_reconnect(rfxtrx, hass: HomeAssistant) -> None:
     """Test that we reconnect on connection loss."""
     config_entry = await setup_rfx_test_cfg(hass, device="/dev/ttyUSBfake")
+    await hass.async_block_till_done()
 
```

Comment:
Should go into the setup helper. Its expected to be complete after that.

Reply:
moved